src/main/java/org/axonframework/intellij/ide/plugin/handler/DefaultEventHandlerRepository.java

```diff
@@ -33,4 +33,26 @@ public void registerHandler(EventHandler eventHandler) {
         }
         return found;
     }
+
+    @Override
+    public Set<EventHandler> findEventHandlers(PsiType eventType) {
```

Comment:
Yes. It will work fine. It's using plugin code which accommodates that. (Also I tested it :))